config/environments/development.rb

```diff
@@ -53,4 +53,18 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_url_options = { host: "localhost", port: "3000" }
+
+   config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    enable_starttls_auto: true,
+    user_name: ENV["GOOGLE_MAIL"],
+    password: ENV["GOOGLE_PASSWORD"],
+    authentication: "plain",
+    openssl_verify_mode: "none",
+    domain: "http://localhost:3000"
+  }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.